libsolidity/interface/StandardCompiler.cpp

```diff
@@ -503,14 +503,16 @@ std::optional<Json::Value> checkMetadataKeys(Json::Value const& _input)
 {
 	if (_input.isObject())
 	{
+		if (_input.isMember("appendCBOR") && !_input["appendCBOR"].isBool())
+			return formatFatalError("JSONError", "\"settings.metadata.append\" must be Boolean");
```

Comment:
```suggestion
			return formatFatalError("JSONError", "\"settings.metadata.appendCBOR\" must be Boolean");
```
If we want to keep it at ``appendCBOR``... still hoping we can find something better...

Reply:
Nice catch. Fixed.